scripts/benchmarks-ci.sh

```diff
@@ -2,10 +2,11 @@
 
 steps=50
 repeat=20
-chainName=$1
+category=$1
+runtimeName=$2
 
-benhcmarkOutput=./parachains/runtimes/$chainName/src/weights
-benhcmarkChainName="$chainName-dev"
+benchmarkOutput=./parachains/runtimes/$category/$runtimeName/src/weights
```

Comment:
This variable is either unused and not required or we miss a rename where it is used.

Reply:
I take it back, just saw it.